Source/Core/VideoCommon/RenderBase.cpp

```diff
@@ -477,13 +488,47 @@ void Renderer::UpdateDrawRectangle(int backbuffer_width, int backbuffer_height)
 	// Check for force-settings and override.
 	if (g_ActiveConfig.iAspectRatio == ASPECT_FORCE_16_9)
```

Comment:
I think it would be more clear to reformat this into switch statement. or an if statement with only two cases.